app/src/lib/utils/string.ts

```diff
@@ -11,3 +11,17 @@ export function hashCode(s: string) {
 	}
 	return hash.toString();
 }
+
+/**
+ * Checks if a string is a single character.
+ */
```

Comment:
Same line of thought as earlier, let's drop these comments since it feels redundant.

Reply:
Will do 👍